sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -86,3 +86,533 @@
 --    behavior: can take a long time.
 
 UPDATE `background_services` SET `require_once`='/library/MedEx/MedEx_background.php' WHERE `name`='MedEx';
+
+#IfNotTable form_eye_base
```

Comment:
Place this entire block of new code behind the above #IfNotTable form_eye_base. Then it will all happen in one swoop only once (including the migration).

Reply:
But this is not eye_form it is the old MedEx pointer that was added with the wrong case.  It needs to happen even if the form_eye_base is there?

Reply:
To clarify, is form_eye_base a new table?

Reply:
YES

Reply:
So, this is all one nuclear step then, correct (new tables get added, old table data gets migrated, old table is moved to backup table)? If that is the case, then it should all be behind just one If statement since it will either all go or not go (ie. there is no in between).

Reply:
So, for example, this entire codeblock could be enclosed by:
```
#IfTable form_eye_mag
```